src/main/rx/crsf.c

```diff
@@ -170,8 +175,10 @@ static void handleCrsfLinkStatisticsFrame(const crsfLinkStatistics_t* statsPtr,
 
 #ifdef USE_RX_LINK_QUALITY_INFO
     if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) {
+        const uint16_t crsfPowerInMw = uplinkTXPowerStates[stats.uplink_TX_Power];
```

Comment:
This is risky and could lead to an array overflow if additional states appear in later CRSF firmware.

Reply:
Nice catch!